WordPress/Sniff.php

```diff
@@ -63,6 +63,8 @@ abstract class Sniff implements PHPCS_Sniff {
 	 */
 	protected $escapingFunctions = array(
 		'absint'                     => true,
+		'count'                      => true,
```

Comment:
This one is already present on https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/WordPress/Sniff.php#L111.

I think the other functions added here should also be on the other list, as the `$escapingFunctions` list does hold the functions whose context is escaping - that is, they are, by design, intended for escaping/sanitizing use. Whereas functions like `count`, `ceil`, `rand`, etc. are not used for escaping but for specific usage (counting, rounding, random number generation, etc.).